test.disabled/ejabberd_tests/tests/push_SUITE.erl

```diff
@@ -0,0 +1,715 @@
+    rpc(code, load_binary, [?MODULE, Filename, Binary]),
+    escalus:init_per_suite(Config).
+end_per_suite(Config) ->
+    escalus:end_per_suite(Config).
+
+init_per_group(disco, Config) ->
+    escalus:create_users(Config, escalus:get_users([alice]));
+init_per_group(muclight_msg_notifications, Config) ->
+    Host = ct:get_config({hosts, mim, domain}),
+    dynamic_modules:start(Host, mod_push, ?PUSH_OPTS),
+    dynamic_modules:start(<<"localhost">>, mod_muc_light,
+                          [{host, binary_to_list(?MUCHOST)},
+                           {backend, mnesia},
+                           {rooms_in_rosters, true}]),
+    Config;
+init_per_group(_, Config0) ->
+    Config = [{push_config, ?PUSH_OPTS} | Config0],
+    Host = ct:get_config({hosts, mim, domain}),
+    dynamic_modules:start(Host, mod_push, ?PUSH_OPTS),
+    escalus:create_users(Config, escalus:get_users([bob, alice])).
+
+end_per_group(disco, Config) ->
+    escalus:delete_users(Config, escalus:get_users([alice]));
+end_per_group(muclight_msg_notifications, _Config) ->
+    Host = ct:get_config({hosts, mim, domain}),
+    dynamic_modules:stop(Host, mod_push),
+    dynamic_modules:stop(Host, mod_muc_light);
+end_per_group(_, Config) ->
+    Host = ct:get_config({hosts, mim, domain}),
+    dynamic_modules:stop(Host, mod_push),
+    escalus:delete_users(Config, escalus:get_users([bob, alice])).
+
+init_per_testcase(CaseName = push_notifications_listed_disco_when_available, Config) ->
+    Host = ct:get_config({hosts, mim, domain}),
+    dynamic_modules:start(Host, mod_push, ?PUSH_OPTS),
+    escalus:init_per_testcase(CaseName, Config);
+init_per_testcase(CaseName = push_notifications_not_listed_disco_when_not_available, Config) ->
+    escalus:init_per_testcase(CaseName, Config);
+init_per_testcase(CaseName, Config) ->
+    start_publish_listener(Config),
+    rpc(mod_muc_light_db_backend, force_clear, []),
+    escalus:init_per_testcase(CaseName, Config).
+
+
+end_per_testcase(CaseName = push_notifications_listed_disco_when_available, Config) ->
+    Host = ct:get_config({hosts, mim, domain}),
+    dynamic_modules:stop(Host, mod_push),
+    escalus:end_per_testcase(CaseName, Config);
+end_per_testcase(CaseName = push_notifications_not_listed_disco_when_not_available, Config) ->
+    escalus:end_per_testcase(CaseName, Config);
+end_per_testcase(CaseName, Config) ->
+    rpc(meck, unload, []),
+    escalus:end_per_testcase(CaseName, Config).
+
+%%--------------------------------------------------------------------
+%% GROUP disco
+%%--------------------------------------------------------------------
+
+push_notifications_listed_disco_when_available(Config) ->
+    escalus:story(
+        Config, [{alice, 1}],
+        fun(Alice) ->
+            Server = escalus_client:server(Alice),
+            escalus:send(Alice, escalus_stanza:disco_info(Server)),
+            Stanza = escalus:wait_for_stanza(Alice),
+            try
```

Comment:
Those checks in other suite were probably implemented when escalus wasn't reporting full stanza log upon error yet. Now there is no need to manually log stanza. Which suite is it BTW?